mackerel-plugin-accesslog/lib/accesslog.go

```diff
@@ -0,0 +1,183 @@
+package mpaccesslog
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/Songmu/axslogparser"
```

Comment:
It is not as simple as you think. The axslogparser covers most cases. Which support following case.

- common log
- common log with vhost
- combined log
- combined log with extended fields
- used tab as separator in apache log
- ltsv accesslog

I should doument it...

Of course I am planning to be able to specify a custom parser later.